zeppelin-interpreter-shaded/pom.xml

```diff
@@ -117,6 +117,8 @@
                 <exclude>org/w3c/dom/**/*</exclude>
                 <exclude>org/xml/sax/*</exclude>
                 <exclude>org/xml/sax/**/*</exclude>
+
+                <exclude>org/eclipse/sisu/**</exclude>
```

Comment:
It's because

```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] Bad service configuration file, or exception thrown while constructing Processor object: javax.annotation.processing.Processor: jar:file:/Users/jl/Works/Projects/zeppelin/zeppelin-interpreter-shaded/target/zeppelin-interpreter-shaded-0.11.0-SNAPSHOT.jar!/META-INF/services/javax.annotation.processing.Processor:1: Illegal provider-class name: org/apache/zeppelin/shaded/org.apache.zeppelin.shaded.org.eclipse.sisu.space.SisuIndexAPT6
```